src/contexts/ThemeContext/ThemeContext.jsx

```diff
@@ -0,0 +1,43 @@
+import { useContext, createContext, useState, useEffect } from "react";
+
+const defaultTheme = { theme: "light" };
+
+const ThemeContext = createContext(defaultTheme);
+
+const ThemeProvider = ({ children }) => {
```

Comment:
Like the implementation of Dark and Light Theme..

Reply:
Thank you for review